Sources/Icon/IconAddress.swift

```diff
@@ -0,0 +1,63 @@
+// Copyright © 2017-2018 Trust.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+import Foundation
+
+/// Icon address.
+public struct IconAddress: Address, Hashable {
+    public static let size = 20
+    public static let prefix = "hx"
+
+    /// Validates that the raw data is a valid address.
+    static public func isValid(data: Data) -> Bool {
+        return data.count == IconAddress.size
+    }
+
+    /// Validates that the string is a valid address.
+    static public func isValid(string: String) -> Bool {
+        let pattern = "^(hx[a-zA-Z0-9]{40})$"
```

Comment:
Do we really need regex to validate the address format?

Reply:
No we don't need the regex to validate, we can check the prefix and the size of the rest of the address like it is done for Ethereum Address. I will change that